elytron/src/main/java/org/wildfly/extension/elytron/KeyStoreService.java

```diff
@@ -271,6 +266,21 @@ boolean isModified() {
         return trackingKeyStore.isModified();
     }
 
+    char[] resolveKeyPassword(final ExceptionSupplier<CredentialSource, Exception> keyPasswordCredentialSourceSupplier) throws Exception {
```

Comment:
Not related to this PR but we also want at some point to look at KeyStore password re-use for the KeyManager resource as well.